src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected static String[] tasks = new String[100];
+    protected static boolean[] tasksCompleted = new boolean[100];
```

Comment:
For this line, I may suggest you to refactor `Task` Class to have a boolean variable `isDone`. As a new `Task` is created, that `isDone` variable is unique to that task and may be updated as required. This is also efficient as you don't have to maintain an extra `Array`. 

Sometimes it may happen that, you delete a `Task` and you may forget to update it in this other `Array`, making your code fragile and more bug prone.

```Java
    protected boolean isTaskDone;
```